textile/fabric_printing/doctype/print_order/print_order.py

```diff
@@ -873,8 +873,8 @@ def _create_design_items_and_boms(self, publish_progress=True, ignore_version=Tr
 				bom_doc = self.make_design_bom(d)
 				bom_doc.flags.ignore_version = ignore_version
 				bom_doc.flags.ignore_feed = ignore_feed
-				bom_doc.flags.ignore_permissions = True
-				bom_doc.save()
+				bom_doc.flags.ignore_permissions = ignore_permissions
```

Comment:
This should always be True. Check permissions separately before creating document